### ISSUE TITLE
towerp0/incorrect_vector:0

### ISSUE THREAD

**Issue by prashantsingh905** [digital-thinking/udacity-nanodegree#1]
I am trying to train the capstone project on my own pc. I have the latest version of tensorpack and i have made all the necessary change but still i am getting this error which i am not able to figure out .
`Traceback (most recent call last):
  File "main.py", line 115, in <module>
    SimpleTrainer(config).train()
  File "/usr/local/lib/python2.7/dist-packages/tensorpack/train/base.py", line 93, in train
    self.setup()
  File "/usr/local/lib/python2.7/dist-packages/tensorpack/train/base.py", line 118, in setup
    self._callbacks.setup_graph(weakref.proxy(self))
  File "/usr/local/lib/python2.7/dist-packages/tensorpack/callbacks/base.py", line 44, in setup_graph
    self._setup_graph()
  File "/usr/local/lib/python2.7/dist-packages/tensorpack/callbacks/group.py", line 77, in _setup_graph
    cb.setup_graph(self.trainer)
  File "/usr/local/lib/python2.7/dist-packages/tensorpack/callbacks/base.py", line 44, in setup_graph
    self._setup_graph()
  File "/usr/local/lib/python2.7/dist-packages/tensorpack/callbacks/inference_runner.py", line 116, in _setup_graph
    self._hooks = [self._build_hook(inf) for inf in self.infs]
  File "/usr/local/lib/python2.7/dist-packages/tensorpack/callbacks/inference_runner.py", line 178, in _build_hook
    fetches = self._get_tensors_maybe_in_tower(out_names)
  File "/usr/local/lib/python2.7/dist-packages/tensorpack/callbacks/inference_runner.py", line 125, in _get_tensors_maybe_in_tower
    return get_tensor_fn(placeholder_names, names, 0, prefix=self._prefix)
  File "/usr/local/lib/python2.7/dist-packages/tensorpack/predict/base.py", line 212, in get_tensors_maybe_in_tower
    tensors = get_tensors_by_names(names)
  File "/usr/local/lib/python2.7/dist-packages/tensorpack/tfutils/common.py", line 112, in get_tensors_by_names
    ret.append(G.get_tensor_by_name(varn))
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/ops.py", line 2554, in get_tensor_by_name
    return self.as_graph_element(name, allow_tensor=True, allow_operation=False)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/ops.py", line 2405, in as_graph_element
    return self._as_graph_element_locked(obj, allow_tensor, allow_operation)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/ops.py", line 2447, in _as_graph_element_locked
    "graph." % (repr(name), repr(op_name)))
KeyError: "The name 'towerp0/incorrect_vector:0' refers to a Tensor which does not exist. The operation, 'towerp0/incorrect_vector', does not exist in the graph.`

**Comment by digital-thinking** [comment]
Hi prashantsingh905,
yes i noticed something like this too, its related to the naming of the tensors in tensorpack as far as I remember. I used a older tensorpack release, which did use other names internally. I am not sure, but i think I fixed that in the windows branch somehow, but i am not sure. The issue was related to the name of the error function. Hope that helps. 

Ohh, better use the real project:
https://github.com/digital-thinking/deep-posemachine
